app/src/components/account/UserMenu.vue

```diff
@@ -168,7 +168,7 @@ export default {
             ...address,
             sessionType: address.sessionType
               ? address.sessionType
-              : address.type || "explore",
+              : "explore",
```

Comment:
I think some address have a `type` field instead of a `sessionType`. Maybe only legacy addresses

Reply:
no, I map them all to sessionType